Error detected but no issues

When running `check-es-compat` it reports:

```
dist/registry.js
  10:50  error  ES2020 optional chaining is forbidden  ecmascript-compat/compat
  11:50  error  ES2020 optional chaining is forbidden  ecmascript-compat/compat

✖ 2 problems (2 errors, 0 warnings)

No issues found. Files are compatible with the target runtimes.
```

So there are 2 errors but no issues found? The return code is also 0 ("ok")

--- a/packages/check-es-compat/bin/cli.mjs
+++ b/packages/check-es-compat/bin/cli.mjs
@@ -44,5 +44,5 @@ async function execute(files) {
   const formatter = await eslint.loadFormatter();
   console.log(formatter.format(results));
 
-  return { hasErrors: results.errorCount > 0 };
+  return { hasErrors: results.some((result) => result.errorCount > 0) };
 }

